pkgs/development/python-modules/owslib/default.nix

```diff
@@ -2,41 +2,39 @@
   lib,
   buildPythonPackage,
   fetchFromGitHub,
-
   lxml,
-  pyproj,
   pytestCheckHook,
   python-dateutil,
   pythonOlder,
-  pytz,
   pyyaml,
   requests,
+  setuptools,
 }:
 
 buildPythonPackage rec {
   pname = "owslib";
-  version = "0.31.0";
-  format = "setuptools";
+  version = "0.32.0";
+  pyproject = true;
 
-  disabled = pythonOlder "3.7";
+  disabled = pythonOlder "3.10";
 
   src = fetchFromGitHub {
     owner = "geopython";
     repo = "OWSLib";
-    rev = version;
-    hash = "sha256-vjJsLavVOqTTrVtYbtA0G+nl0HanKeGtzNFFj92Frw8=";
+    rev = "refs/tags/${version}";
+    hash = "sha256-q2O9FNBszNWfL1ekcohSd1RbdLFu8c+zxi+UFeQ7/mk=";
   };
 
   postPatch = ''
```

Comment:
Please remove postPatch and use pytest-cov-stub instead.

Reply:
The `postPatch` section can not be removed as we still need to get rid to the doctests.